srcpkgs/keepassxc/template

```diff
@@ -1,45 +1,28 @@
 # Template file for 'keepassxc'
 pkgname=keepassxc
-version=2.3.4
+version=2.4.0
 revision=1
 build_style=cmake
-configure_args="-DWITH_TESTS=OFF
- -DWITH_XC_AUTOTYPE=$(vopt_if autotype ON OFF)
- -DWITH_XC_NETWORKING=$(vopt_if network ON OFF)
- -DWITH_XC_YUBIKEY=$(vopt_if yubikey ON OFF)
- -DWITH_XC_BROWSER=$(vopt_if browser ON OFF)
- -DWITH_XC_SSHAGENT=$(vopt_if sshagent ON OFF)"
-makedepends="qt5-tools-devel libgcrypt-devel libargon2-devel
-	$(vopt_if autotype 'qt5-x11extras-devel libXtst-devel libXi-devel')
-	$(vopt_if browser libsodium-devel)
-	$(vopt_if network libcurl-devel)
-	$(vopt_if yubikey 'libykpers-devel libyubikey-devel')"
+configure_args="-DWITH_TESTS=OFF -DWITH_XC_ALL=ON"
+makedepends="qt5-tools-devel qt5-svg-devel libgcrypt-devel libargon2-devel qrencode-devel
+	qt5-x11extras-devel libXtst-devel libykpers-devel libyubikey-devel libsodium-devel"
 short_desc="KeePassXC is a cross-platform port of “Keepass Password Safe”"
 maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
-license="GPL-3.0-only"
+license="GPL-3.0-or-later, BSD-3-Clause, CC0-1.0, LGPL-2.0-only, LGPL-2.1-only,
+	LGPL-3.0-or-later, Nokia-Qt-exception-1.1, MIT BSL-1.0"
 homepage="https://keepassxc.org/"
 distfiles="https://github.com/keepassxreboot/keepassxc/releases/download/${version}/keepassxc-${version}-src.tar.xz"
-checksum=0c24a78895185b9b6b098f08a15b332c600f80aa570d608940fcc2a38e02780a
+checksum=081ff1a34da8e0e8db951b5d70824d7a2b3d299ed4ae54e6ddc1e3ac176b95a8
 
 if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" qt5-host-tools qt5-qmake qt5-tools"
+	hostmakedepends+=" qt5-qmake qt5-host-tools"
 fi
 
-build_options="autotype browser network sshagent yubikey"
-desc_option_autotype="Enable auto-type"
-desc_option_browser="Enable browser integration with keepassxc-browser"
-desc_option_network="Enable networking support (favicon download)"
-desc_option_sshagent="Enable SSH agent support"
-desc_option_yubikey="Enable YubiKey HMAC-SHA1 authentication support"
-build_options_default="autotype browser sshagent yubikey network"
-
 post_install() {
```

Comment:
LICESE.BOOST-1.0 COPYING LICENSE.NOKIA-LGPL-EXCEPTION LICENSE.CC0 don't need to be vlicense.

Reply:
done